mangaki/mangaki/wrappers/anilist.py

```diff
@@ -65,91 +65,103 @@ class AniListStatus(Enum):
 
     @property
     def start_date(self) -> Optional[datetime]:
-        if self.anime_info['start_date_fuzzy']:
-            return to_python_datetime(self.anime_info['start_date_fuzzy'])
+        if self.work_info['start_date_fuzzy']:
+            return to_python_datetime(self.work_info['start_date_fuzzy'])
         return None
 
     @property
     def end_date(self) -> Optional[datetime]:
-        if self.anime_info['end_date_fuzzy']:
-            return to_python_datetime(self.anime_info['end_date_fuzzy'])
+        if self.work_info['end_date_fuzzy']:
+            return to_python_datetime(self.work_info['end_date_fuzzy'])
         return None
 
     @property
+    def description(self) -> Optional[str]:
+        return self.work_info.get('description')
+
+    @property
     def synonyms(self) -> List[str]:
-        return list(filter(None, self.anime_info['synonyms']))
+        return list(filter(None, self.work_info['synonyms']))
 
     @property
     def genres(self) -> List[str]:
-        return list(filter(None, self.anime_info['genres']))
+        return list(filter(None, self.work_info['genres']))
 
     @property
     def is_nsfw(self) -> bool:
-        return self.anime_info['adult']
+        return self.work_info['adult']
 
     @property
     def poster_url(self) -> str:
-        return self.anime_info['image_url_lge']
+        return self.work_info['image_url_lge']
 
     @property
     def nb_episodes(self) -> Optional[int]:
         if self.work_type == AniListWorks.animes:
-            return self.anime_info['total_episodes']
+            return self.work_info['total_episodes']
+        return None
+
+    @property
+    def episode_length(self) -> Optional[int]:
+        if self.work_type == AniListWorks.animes:
+            return self.work_info.get('duration')
         return None
 
     @property
     def nb_chapters(self) -> Optional[int]:
         if self.work_type == AniListWorks.mangas:
-            return self.anime_info['total_chapters']
+            return self.work_info['total_chapters']
         return None
 
     @property
     def status(self) -> Optional[AniListStatus]:
         if self.work_type == AniListWorks.animes:
-            return AniListStatus(self.anime_info['airing_status'])
+            return AniListStatus(self.work_info['airing_status'])
         elif self.work_type == AniListWorks.mangas:
-            return AniListStatus(self.anime_info['publishing_status'])
-        else:
-            return None
+            return AniListStatus(self.work_info['publishing_status'])
+        return None
 
     @property
-    def tags(self) -> List[Dict[str, Any]]:
-        return [
-            {
-                'name': tag['name'],
-                'anilist_tag_id': tag['id'],
-                'spoiler': tag['spoiler'] or tag['series_spoiler']
-            } for tag in self.anime_info['tags']
-        ]
+    def tags(self) -> Optional[List[Dict[str, Any]]]:
+        if self.work_info.get('tags'):
```

Comment:
Rather than testing this and doing something complex or long, test the contrary and do an early return.

```python
if 'tags' not in self.work_info:
    return None

# do usual stuff.
```

It improves readability of your code.